File: src/main/kotlin/org/voutixcloud/data/FileWriter.java
package org.voutixcloud.data;

import java.io.File;
import java.io.IOException;

public class FileWriter {

    public static void write(File file, String text) {

        try (java.io.FileWriter writer = new java.io.FileWriter(file)) {

            writer.write(text);
            writer.flush();
            writer.close();

        } catch (IOException e) {

            e.printStackTrace();

        }

    }

}

File: src/main/kotlin/org/voutixcloud/data/FileWriter.kt
package org.voutixcloud.data

import java.io.File
import java.io.FileWriter
import java.io.IOException

class FileWriter {
    internal fun write(file: File, text: String) {

        var writer: java.io.FileWriter = FileWriter(file)

        try {
            writer.write(text)
            writer.flush()
            writer.close()

        } catch (e: IOException) {
            e.printStackTrace()
        }
    }
}